docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc

```diff
@@ -0,0 +1,121 @@
+[id="creating-a-ceph-nfs-cluster_{context}"]
+
+= Creating a Ceph NFS cluster
+
+If you use the Ceph via NFS backend with {CephService_first_ref}, prior to adoption, you must create a new clustered NFS service on the Ceph cluster. This service will replace the standalone, pacemaker-controlled `ceph-nfs` service that was used on {osp_prev_long} {osp_prev_ver}.
```

Comment:
CephService_first_ref <- it's Manila, an OSP service, while `NFS` is the Ceph Service we are using in this guide.